JASP-Desktop/data/importers/importer.h

```diff
@@ -8,9 +8,13 @@
 
 class ImportDataSet;
 class ImportColumn;
+#include <QCoreApplication>
+#include "utilities/qutils.h"
+
 
 class Importer
 {
+	Q_DECLARE_TR_FUNCTIONS(CSVImporter)
```

Comment:
Why CSVImporter? Should it not be Impoter?

Reply:
Strange, I changed it.